python/paddle/v2/fluid/io.py

```diff
@@ -283,25 +325,8 @@ def save_inference_model(dirname,
     with open(model_file_name, "wb") as f:
         f.write(inference_program.desc.serialize_to_string())
 
-    save_params(executor, dirname, main_program)
-
-
-def load_persistables_if_exist(executor, dirname, main_program=None):
-    filenames = next(os.walk(dirname))[2]
-    filenames = set(filenames)
-
-    def _is_presistable_and_exist_(var):
-        if not is_persistable(var):
-            return False
-        else:
-            return var.name in filenames
-
-    load_vars(
-        executor,
-        dirname,
-        main_program=main_program,
-        vars=None,
-        predicate=_is_presistable_and_exist_)
+    parameter_list = get_parameters(inference_program)
+    save_vars(executor, dirname, inference_program, parameter_list)
```

Comment:
#7874 is merged. Directly change of the `save_inference_model` will fail the CI of the develop branch. Please update the develop branch first.